src/homepageExperience/components/steps/Finish.tsx

```diff
@@ -69,30 +69,38 @@ const handleNextStepEvent = (wizardEventName: string, nextStepName: string) => {
   event(`firstMile.${wizardEventName}.nextSteps.${nextStepName}.clicked`)
 }
 
+const marginTopStyle = {marginTop: '80px'}
+
 export const Finish = (props: OwnProps) => {
+  const {
+    finalFeedback,
+    finishStepCompleted,
+    markStepAsCompleted,
+    setFinalFeedback,
+    wizardEventName,
+  } = props
+
   useEffect(() => {
     // if the finish step was opened during the session,
     // this check prevents from multiple logging of finish events
     // in case user navigates back and forth
-    if (!props.finishStepCompleted) {
-      event(`firstMile.${props.wizardEventName}.finished`)
-      props.markStepAsCompleted()
+    if (!finishStepCompleted) {
```

Comment:
Confirming that we tested this with `console.logs`, and does not impact eventing behavior.